fabric/.gitignore

```diff
@@ -3,3 +3,10 @@
 /chaincodes/*.tar.gz
 /crypto-config/
 /channel-artifacts/
+/couchdb
```

Comment:
I think you need to remove the leading `/` here or prefix it with `/fabric`?

Reply:
Looks like this file isn't even generated anymore. Deleted.